pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -178,9 +178,9 @@ stdenv.mkDerivation ({
 
     BINDGEN_CFLAGS="$(< ${stdenv.cc}/nix-support/libc-cflags) \
```

Comment:
Maybe we should just expose all of these flags in the cc-wrapper ? Something like `stdenv.cc.cflags_compile` and `stdenv.cc.ld_flags`

Reply:
that is the command to get it since we don't know the actual values at evaluation

Reply:
It would be nice to uses more Nix so we did know the values. (e.g. have a nix boolean and then assert at run-time it was correct.)

Reply:
I think that requires Nix-level setup-hooks to work correctly. I think that might be too ambitious for now.

Reply:
I'm thinking something like `/nix-support/get-cflags.sh` which does all of this for you.

Reply:
Well getting all the -I flags for other packaged requires that, but not just the std library stuff I think.

Reply:
Right these should just need the basic c/c++ headers and library paths.